resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,385 @@
+openapi: 3.0.3
+info:
+  title: CTC Traders Test Support
+  contact: {}
+  version: '1.0'
+  description: |
+    # Overview
+    This API allows testers to inject departure and arrival movement notifications as 
+    if they have come from the office of departure or the office of destination by the New Computerised Transit System (NCTS).
+servers:
+  - url: https://test-api.service.hmrc.gov.uk
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk
+    description: Production
+paths:
+  /test/customs/transits/movements/arrivals/{arrivalId}/messages:
+    post:
+      tags:
+      - Arrivals
+      summary: Inject a fake NCTS Arrival Message
+      description: >
+        Inject a message type into the test NCTS database for a given Arrival Movement.
+      operationId: postFakeNCTSArrivalMessage
+      parameters:
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          name: arrivalId
+          description: The ID specifying the Arrival
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '1'
+      requestBody:
+        description: JSON with message type
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/TestCustomsTransitsMovementsArrivalsMessagesRequest'
+              - description: JSON with message type
+                example:
+                  message:
+                    messageType: IEXXX
+            example:
+              message:
+                messageType: IEXXX
+        required: true
+      responses:
+        '201':
+          description: When a test message has been successfully injected.

```

Comment:
For other CTC API definitions, we are not using full stops for `2XX` response descriptions. Also, use of "when" looks awkward.

Suggestions:
- A successful test message injection response
- Test message has been injected successfully